scripts/publish-to-npm.js

```diff
@@ -49,7 +49,7 @@ const main = () => {
 const publishPackages = (packages = [], isNightly = false) => {
     verifyCleanWorkingTree()
 
-    const publicPackages = JSON.parse(sh.exec('lerna list --json', {silent: true}))
+    const publicPackages = JSON.parse(sh.exec('npx lerna list --json', {silent: true}))
```

Comment:
This is to make sure that `lerna` is found, and the command can run successfully in all environments (both locally and in CI).